### PR TITLE
removed configsync dependency

### PR DESCRIPTION
**Pull request by henderiw**
Removed the dependency to configsync namespace, but we need to see if the namespace is to be set.

### CODE REVIEW

**Comment by henderiw** [comment]
So this PR adds 2 changes to make it independent on configsync.
1. the syncApp should be changed from configsync to tobeinstalledonremotecluster
2. the remoteNamespace can be set. For Configsync this has to eb set.

**Comment by arora-sagar** [comment]
@henderiw If I understand correctly I still have to create the secret in the management cluster right? Only then Nephio-Controller will act on it. In this case is it possible to add a possibility to populate the secret in the multiple clusters?

File: controllers/pkg/reconcilers/bootstrap-secret/README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # bootstrap secret controller
 
-The bootstrap controller is a k8s controller which goal is to bootstrap secrets on a newly installed cluster. This ensure e.g. that a gitops tool like `config-sync` is installed on a cluster and subsequent configurations tasks get handled through the gitops toolchain.
+The bootstrap controller is a k8s controller which goal is to bootstrap secrets on a newly installed cluster. This ensure e.g. that a gitops tool like `config-sync`, `argo`, `flux` or others are installed on a remote cluster and subsequent configurations tasks get handled through the gitops toolchain.
 
 ## implementation
 
 The controller acts on a secret. It first figures out if the secret is to be installed on the remote cluster, by checking if:
-- annotation key `nephio.org/app` is equal to `configsync`
+- annotation key `nephio.org/app` is equal to `toBeInstalledOnRemoteCluster`
```
[the remark this answers is not in the document]
Comment (henderiw) [reply controllers/pkg/reconcilers/bootstrap-secret/README.md]: all lowercase. made it consistent

File: controllers/pkg/reconcilers/bootstrap-secret/README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 - annotation key `nephio.org/cluster-name` is not an empty string or `mgmt`
```
[the remark this answers is not in the document]
Comment (henderiw) [reply controllers/pkg/reconcilers/bootstrap-secret/README.md]: ok added, but I created a dedicated section

**Comment by henderiw** [comment]
@arora-sagar I added logic to add the secret to a multiple clusters.

File: controllers/pkg/reconcilers/bootstrap-secret/reconciler.go [hunk 3]
```diff
@@ -42,11 +42,14 @@ func init() {
...
-	configsyncApp       = "configsync"
-	bootstrapApp        = "bootstrap"
-	configsyncNamespace = "config-management-system"
+	clusterNameKey     = "nephio.org/cluster-name"
+	nephioAppKey       = "nephio.org/app"
+	remoteNamespaceKey = "nephio.org/remote-namespace"
+	syncApp            = "tobeinstalledonremotecluster"
+	bootstrapApp       = "bootstrap"
+
+	//configsyncApp       = "configsync"
```
[the remark this answers is not in the document]
Comment (henderiw) [reply controllers/pkg/reconcilers/bootstrap-secret/reconciler.go]: fixed

File: controllers/pkg/reconcilers/bootstrap-secret/README.md [hunk 4]
```diff
@@ -1,16 +1,15 @@
 # bootstrap secret controller
 
-The bootstrap controller is a k8s controller which goal is to bootstrap secrets on a newly installed cluster. This ensure e.g. that a gitops tool like `config-sync` is installed on a cluster and subsequent configurations tasks get handled through the gitops toolchain.
+The bootstrap controller is a k8s controller which goal is to bootstrap secrets on a newly installed cluster. This ensure e.g. that a gitops tool like `config-sync`, `argo`, `flux` or others are installed on a remote cluster and subsequent configurations tasks get handled through the gitops toolchain.
 
 ## implementation
```
[the remark this answers is not in the document]
Comment (henderiw) [reply controllers/pkg/reconcilers/bootstrap-secret/README.md]: added an example

**Pull request merged**